src/util/rebind.js

```diff
@@ -34,6 +34,8 @@ function capitalizeFirstLetter(str) {
  * be rebound.
  */
 export function rebindAll(target, source, prefix, exclusions) {
+    prefix = typeof prefix !== 'undefined' ? prefix : '';
```

Comment:
No, not a bad merge, just poor memory. This allows properties to be re-bound without a prefix, as tested here:

https://github.com/ScottLogic/d3fc/pull/600/files#diff-f6d6160b813bdc9d086071b5af5c3e5eR73